src/containers/SideMenu.js

```diff
@@ -155,7 +155,7 @@ const styles = StyleSheet.create({
   sceneContainer: {
     flex: 1,
     marginLeft: -15,
-    flexDirection: 'row'
```

Comment:
weird that line isnt in my project ill leave it out for the next merge and never got the linear gradient to work.  when i had it set to true there would be no margin but when it was false i got the margin on the side menu